src/reducers/index.ts

```diff
@@ -164,7 +151,7 @@ function parseVega(
   currLogger.level(state.logLevel);
 
   try {
-    const spec = parseJSONC(action.spec);
+    const spec = parseJSONCOrThrow(action.spec);
```

Comment:
Makes me even more convinced that snake case is better than camel case. Oh well.